src/libostree/ostree-repo-finder-mount.c

```diff
@@ -413,15 +414,20 @@ ostree_repo_finder_mount_resolve_async (OstreeRepoFinder                  *finde
       /* Sort the repos lexically. */
       g_array_sort (repos_refs, repo_and_refs_compare);
 
-      /* Also check the .ostree/repo and ostree/repo directories in the mount,
-       * as well-known special cases. Add them after sorting, so they’re always
-       * last. */
-      scan_and_add_repo (mount_root_dfd, ".ostree/repo", FALSE,
-                         mount_name, &mount_root_stbuf,
-                         parent_repo, repos_refs, cancellable);
-      scan_and_add_repo (mount_root_dfd, "ostree/repo", FALSE,
-                         mount_name, &mount_root_stbuf,
-                         parent_repo, repos_refs, cancellable);
+      /* Also check the well-known special-case directories in the mount.
+       * Add them after sorting, so they’re always last. */
+      const gchar * const well_known_repos[] =
+        {
+          ".ostree/repo",
+          "ostree/repo",
+          "var/lib/flatpak",
+          "var/lib/containers/atomic",
```

Comment:
Sorry I was wrong when I mentioned this; it uses that dir for checkouts but actually uses `/ostree/repo`.

Reply:
(Otherwise LGTM, can you force-push an update that also drops that bit from the commit msg title?  Sorry again for the incorrect info originally)

Reply:
Force-pushed an update.